cloud-optimized-geotiffs/accessing-cogs-in-r-terra.ipynb

```diff
@@ -0,0 +1,526 @@
+{
```

Comment:
<p>we should switch to either using terra::describe or sf::gdal_utils("info") which are R installable and do not rely on a system install of gdal</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>